Makefile

```diff
@@ -79,6 +78,9 @@ libflux: libflux/target/debug/libflux.a
 libflux/target/debug/libflux.a:
 	cd libflux && $(CARGO) build $(CARGO_ARGS)
 
+libflux/go/libflux/flux.h: libflux/include/influxdata/flux.h
+	$(GO_GENERATE) ./libflux/go/libflux
```

Comment:
I think I missed some context here, why did you add this extra generate step?

Reply:
The reason is far from  being trivial.

`flux.h` was in an `include` dir by itself, we needed it to be alongside `.go` files to get it exported if vendored when building (because that's how `go mod vendor` works like that). So, we added this other generation step that copies over `flux.h` in a dir where there are `.go` files. We decided to copy, and not move it, because having an `include` dir states out clearly which are the headers of the lib.

Hope this clears things out